### DISCUSSION TITLE
Chrome rendering autocomplete list with autocomplete="off"

### DISCUSSION THREAD

**Discussion by jhdielman** [twilio-labs/paste#1375] [General] [answered]
Recently I noticed that Chrome's autocomplete list appears even with autocomplete set to "off". Is there another way around this?

**Comment by zahnster** [comment, marked as answer]
There appears to be a longstanding issue in Chrome that ignores the autocomplete attribute. I found some information in this Chromium thread:
https://bugs.chromium.org/p/chromium/issues/detail?id=914451&can=2&q=Autofill%20&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified

It does not appear at this time that there is an acceptable workaround.

**Reply by TheSisb** [reply]
There are some ideas in [this SO thread](https://stackoverflow.com/questions/60689757/how-to-disable-chrome-autofill-after-2020), but I haven't tested them myself.  tl;dr: use a unique name, id, and autocomplete value.
